packages/stark-ui/src/modules/session-ui/components/session-card/session-card.component.html

```diff
@@ -0,0 +1,13 @@
+<mat-card>
```

Comment:
Before, in every session page we had `role="alertdialog" aria-busy="true" aria-live="assertive"`. Is this not relevant anymore? Maybe it is still relevant right?

Reply:
I added back some ARIA attributes that seemed relevant.